vignettes/articles/03-run-stress-test.Rmd

```diff
@@ -233,4 +235,4 @@ given calculation.
 ## Interpretation of the output files
 
 Detailed explanations of the result files can be found in the vignette:
-["Understand the output files of the transition risk stress test"](https://2degreesinvesting.github.io/r2dii.climate.stress.test/articles/articles/04-read-the-outputs.html).
+[Understand the output files of the transition risk stress test](https://2degreesinvesting.github.io/r2dii.climate.stress.test/articles/articles/04-read-the-outputs.html).
```

Comment:
this link does not work anymore, because the number is now 05 outputs, not 04.
please check the links are still correct

Reply:
thanks, I checked all links